app.js

```diff
@@ -2,24 +2,20 @@
 //  core
 //
 
-const bus = require('./bus.js');
+const bus = require('./bus');
+const bots = require('./bots');
 
-const __DEV__ = process.env.NODE_ENV === 'dev';
+const botsList = Object.keys(bots).map(botName => bots[botName]);
 
 // global for debugging
-const jabber = global.jabber = require('./bots/jabber');
-const telegram = global.telegram = require('./bots/telegram');
+Object.keys(bots).forEach(k => {
+  global[k] = bots[k];
+});
```

Comment:
Do we really need to store bots in global variables? I can't see where are they used.

Reply:
Already discussed and deleted those globals